regulations/uitests/diff_test.py

```diff
@@ -29,6 +30,24 @@ def test_diffs(self):
         # drawer button should be active
         self.assertTrue('current' in drawer_button.get_attribute('class'))
 
+        # drawer should display regulation history
+        timeline_list_items = self.driver.find_elements_by_class_name('history-toc-list')
+        expected_timeline_versions = [
+            '2011-11111',
+            '2012-12121'
+        ]
+        for index, item in enumerate(timeline_list_items):
```

Comment:
It might be better to write this test backwards, so iterate over the expected output and confirm that it matches what you get. Here you iterate over the output you see, but let's say you didn't get back any `.history-toc-list` elements, this test would still pass incorrectly.

Reply:
GOOD CATCH! It turns out that, thanks to a dumb mistake I made with the selector, this exact thing was happening, and the test was passing despite an empty `timeline_list_items` list. This should be fixed (and now correctly passing) in 6381fa7eba9dc31ad2da21ab50f01339ecd78574.